.github/workflows/pr.yml

```
@@ -81,10 +81,13 @@ jobs:
           set -x
           git checkout -b branch
           git fetch --unshallow origin main
-          cmd="ansible-test sanity --verbose --docker --color --coverage --failure-ok"
-          $cmd 2>&1 | tee branch.output
+          for version in 3.6 3.7 3.8 3.9 ; do
+            ansible-test sanity --verbose --docker --python $version --color --coverage --failure-ok
+          done 2>&1 | tee branch.output
```

Only 3.9 is being kept in the output, you want to append instead
```suggestion
          done 2>&1 | tee -a branch.output
```